package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lightful/ray",
-  "version": "3.0.2",
+  "version": "3.0.3",
```

Comment:
You don't have to change the version here. It's all handled by npm when you run `yarn release` after merging in master